tests/crypto/main.cpp

```diff
@@ -99,7 +99,7 @@ int main(int argc, char *argv[]) {
       vector<char> data;
       ec_scalar expected, actual;
       get(input, data, expected);
-      hash_to_scalar(data.data(), data.size(), actual);
+      ::hash_to_scalar(data.data(), data.size(), actual);
```

Comment:
Yes, because I defined the function `crypto::hash_to_scalar()` in crypto.h and crypto-tests.h has a global scope function `::hash_to_scalar()`, so the compiler gets confused which to use due to `using namespace crypto`.